HISTORY.md

```diff
@@ -3,6 +3,9 @@
   * Add additional L-BFGS convergence check
     ([#136](https://github.com/mlpack/ensmallen/pull/136)).
 
+  * Add AdaBound and AMSBound
+    ([#137](https://github.com/mlpack/ensmallen/pull/137)).
```

Comment:
I think we'll need to rebase this on current master---easy enough to do during merge.